WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1 +1,3 @@
-* [*] Block Editor: Update link colors in action sheets from green to blue [https://github.com/WordPress/gutenberg/pull/42996]
+Links in action sheets (those pop-up menus on your device that bring up extra options) are now blue instead of green.
```

Comment:
Sorry for missing this earlier and before this was sent for editorial, but looking at https://github.com/WordPress/gutenberg/pull/42996 it seems that this change in Gutenberg might only after iOS after all and this shouldn't have been added to the Android release notes?

cc @SiobhyB @dcalhoun ?

Reply:
And in any case, this entry in the release notes for Jetpack felt odd, because it's about Jetpack and yet we're saying we updated colors from green to blue, not the other way around…?!

Reply:
For posterity, just x-posting the comment here to confirm all looks good with the notes: https://github.com/wordpress-mobile/WordPress-Android/pull/17145#discussion_r965703697